games-arcade/rocksndiamonds/rocksndiamonds-3.3.1.2.recipe

```diff
@@ -59,5 +62,9 @@ INSTALL()
 	destdir=$appsDir/"Rocks'n'Diamonds"
 	mkdir -p $destdir
 	cp -r rocksndiamonds sounds graphics levels music $destdir
+
```

Comment:
You should add the versioning info here, this should do the trick:
	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
	local MINOR="`echo "$portVersion" | cut -d. -f3`"
	local INTERNAL="`echo "$portVersion" | cut -d. -f4`"
	sed \
		-e "s|@MAJOR@|$MAJOR|" \
		-e "s|@MIDDLE@|$MIDDLE|" \
		-e "s|@MINOR@|$MINOR|" \
		-e "s|@INTERNAL@|$INTERNAL|" \
		$portDir/additional-files/rocksndiamonds.rdef > rocksndiamonds.rdef